lib/src/dynamics/joints/friction_joint_def.dart

```diff
@@ -8,7 +8,7 @@ class FrictionJointDef<A extends Body, B extends Body> extends JointDef<A, B> {
   /// The maximum friction torque in N-m.
   double maxTorque = 0.0;
 
-  FrictionJointDef() : super(JointType.friction);
+  FrictionJointDef() : super();
```

Comment:
@spydon what do we think of removing the constructor altogether here?

Reply:
Sgtm!

Reply:
Perfect, in the future, when we improve documentation we might require to add it to add the `{@macro}`, to support IDE highlighting the doc when hovered. For the time being, I think it's not required.